test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -424,7 +402,7 @@ export default class PerfCompare extends Component {
             }
         )
         let curAllVariantData = [];
-        let curVariantData, curMetricTable, curMatchingTestVariantIndex, curMatchingTestMetricIndex;
+        let curVariantData, curMetricTable, curMatchingTestVariantIndex, curMatchingTestMetricIndex, metricProps;
```

Comment:
What is `metricProps ` for?

Reply:
Sorry, it's a duplicate definition here(deleted it already). Please see import on line 9 and its usage on line 477.  It's for comparing results in the benchmarks metrics, basically for deciding `highbetter` or not.

Reply:
it should not be declared here as it is declared at line 477.

Reply:
> it should not be declared here as it is declared at line 477.

Yes, I agree, and it has been deleted in my update.